recipe/meta.yaml

```diff
@@ -12,19 +12,21 @@ source:
   sha256: abba8b5ce728520c3a0f1535eab19eac3c14aeef7faa5aded90017ceac2711d3
 
 build:
-  number: 1
+  number: 2
   skip: True  # [win]
   detect_binary_files_with_prefix: true
 
 requirements:
   build:
-    - perl
+    - perl 5.22.2.1
     - pkg-config
     - harfbuzz 1.3.*
     # NOTE: harfbuzz installs everything that pango needs, but to avoid issues
     # with defaults we need to pin against conda-forge versions.
     - fontconfig 2.12.*
     - freetype 2.7.*
+    - gobject-introspection  # [not win]
+    - python >=2.7,<3  # [not win]
```

Comment:
Yeah, I've received conflicting advice on this front. I guess I feel that it's a bit better to err on the side of more redundancy rather than less.